bool grasp_state
string object_name
float64 grasp_size
---
int64 execute_state